<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DL</class>
 <widget class="QMainWindow" name="DL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>895</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Deep Learning Comparator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="container">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1061</width>
      <height>831</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1061</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1061</width>
      <height>831</height>
     </size>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>110</y>
       <width>1041</width>
       <height>621</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1041</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1041</width>
       <height>621</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="feed_container">
      <item>
       <widget class="QLabel" name="yolo_v3">
        <property name="minimumSize">
         <size>
          <width>517</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>517</width>
          <height>619</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>icon/yolov3.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="faster_rcnn">
        <property name="minimumSize">
         <size>
          <width>517</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>517</width>
          <height>619</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::WinPanel</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>icon/rcnn.jpg</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>338</x>
       <y>740</y>
       <width>421</width>
       <height>29</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>421</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>421</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="buttons_layout">
      <item>
       <widget class="QPushButton" name="open_btn">
        <property name="minimumSize">
         <size>
          <width>133</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Open File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="start_btn">
        <property name="minimumSize">
         <size>
          <width>133</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Start Comparision</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="stop_btn">
        <property name="minimumSize">
         <size>
          <width>133</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>740</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="radio_layout">
      <item>
       <widget class="QRadioButton" name="live_camera">
        <property name="minimumSize">
         <size>
          <width>158</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>158</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Live Camera</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="video_file">
        <property name="minimumSize">
         <size>
          <width>158</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>158</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Video File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="image_file">
        <property name="minimumSize">
         <size>
          <width>158</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>158</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Image File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>890</x>
       <y>740</y>
       <width>160</width>
       <height>80</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="exit_layout">
      <item>
       <widget class="QPushButton" name="exit_btn">
        <property name="minimumSize">
         <size>
          <width>158</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>158</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="algo_title_frame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>1041</width>
       <height>61</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>1041</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1041</width>
       <height>61</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="yolo_label">
        <property name="minimumSize">
         <size>
          <width>516</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>516</width>
          <height>57</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Yolo-V3</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="frcnn_label">
        <property name="minimumSize">
         <size>
          <width>516</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>516</width>
          <height>57</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>DejaVu Sans</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Faster R-CNN</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>-1</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>790</y>
       <width>421</width>
       <height>29</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>421</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>421</width>
       <height>80</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="buttons_layout_2">
      <item>
       <widget class="QPushButton" name="recorder">
        <property name="minimumSize">
         <size>
          <width>133</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Record Video</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="downloader">
        <property name="minimumSize">
         <size>
          <width>133</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>133</width>
          <height>27</height>
         </size>
        </property>
        <property name="text">
         <string>Save Image</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1078</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
